### DISCUSSION TITLE
Something wrong in VTEC parametrization

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#252] [Q&A]
Hello developers! 
Recently I've carefully read VTEC parts of your handbook and strasser's PhD thesis. Suspiciously, I found GROOPS only set station-relative VTEC parameters, which means VTEC is independant with satellite location. However, as far as I know, VTEC is dependant with pierce point under the single layer model, pierce point varis with satellite location so that VTEC do correlate with satellite location. Having that in mind, I'm not sure the GROOPS get the right VTEC solution, because VTEC should set for each pierce point (frankly, VTEC should set with each receiver and satellite pair for each epoch). Limited by my  humble knowledge, I could not found clues to explain why GROOPS solve VTEC in that way. Could you give me some suggestion about this? Any explanation will be of great help! 
regards,
Yong F

**Comment by tmayerguerr** [comment]
Hi Yong F,
GROOPS provides two ways to estimate VTEC: One is a global [IonosphereMap](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#ionosphereMap) estimated from a global network. This one assumes a single layer model and compute the pierce point for each satellite-receiver direction correctly. If you do not use a global network, but still want to separate TEC and biases, you can use the simplified [IonosphereVTEC](https://groops-devs.github.io/groops/html/gnssParametrizationType.html#ionosphereVTEC) which assumes a homogeneous ionosphere layer around the station (independent of the pirce point).
regards
Torsten

**Reply by FFFFFG-IGG** [reply]
Hi Torsen
I‘m not interested in seperating Code bias but want to get the biased STEC. In the case of regional network, spherical harmonic is obviously not appropriate. So I'd like to know if the STEC output in residuals is the right value - biased with code bias in the case of not estimating VTEC?
regards
Yong F

**Reply by FFFFFG-IGG** [reply]
In short, I'd like to get unbiased STEC, how to realize this?

**Comment by tmayerguerr** [comment]
Hi Yong F,

Sorry, it was not clear to me from your question that it was about regional TEC estimation. A regional VTEC estimate has not yet been implemented.

For an unbiased STEC estimate, you should set the following [parametrizations](https://groops-devs.github.io/groops/html/gnssParametrizationType.html):
- loosely constrained IonosphereSTEC
- IonosphereVTEC
- TecBiases

The STEC values are included in the residual files ([processingStep->writeResiduals](https://groops-devs.github.io/groops/html/gnssProcessingStepType.html#writeResiduals)) as [type](https://groops-devs.github.io/groops/html/gnssType.html) `I***`.

best regards
Torsten

**Reply by FFFFFG-IGG** [reply]
Thanks for your reply! I'd like to know whether the unbiased STEC estimated from your recommandation is feasible and can be used in STEC modeling?  I would be appreciate if you provide your related work (published papers) about the feasibility of STEC solution. Many thanks!
